lib/wpc/slip.c

```diff
@@ -138,13 +139,13 @@ static uint32_t slip_encode_buffer(uint8_t * buffer,
         if (current == END_SLIP_OCTET || current == ESC_SLIP_OCTET) {
             //Check for buffer_escaped overflow
             if (write >= len_escaped)
-                return -1;
+                return -WPC_INT_WRONG_BUFFER_SIZE;
```

Comment:
Why can't the enum be negative instead?

Reply:
I changed it. It is a bit cleaner this way